dist/access_check/MODULARITY.md

```diff
@@ -0,0 +1,44 @@
+# Rules for Modularity
```

Comment:
I like the content too. However it would be fair to show how the script can be used (i.e any specific option) and outputs to have an idea what errors may look like.

Reply:
Sure, but we need to have a something to start from. In Q4, how do we show people what to expect from this script? What do they need to solve and what is ok to leave behind?

Reply:
That's going to be a part of the Playbook or will be referred from it.